tests/testthat/test-align.R

```diff
@@ -14,3 +14,40 @@ test_that("align helpers [PMT-TEST-0001]", {
   ali <- cols_center(FOO = 'l', bar = 'r')
   expect_identical(ali$update, list(FOO = 'l', bar = 'r'))
 })
+
+test_that("update align objects", {
+  a <- cols_align(A = "l", .r = "B,C,D", .default = "c")
+  expect_is(a, "aligncol")
+
+  b <- update(a, .default = "l")
+  expect_is(b, "aligncol")
+  expect_identical(a$default, "c")
+  expect_identical(b$default, "l")
+
+  b <- update(a, .coltype = "m")
+  expect_is(b, "aligncol")
+  expect_identical(a$coltype, "p")
+  expect_identical(b$coltype, "m")
+
+  b <- update(a, .outer = "lr")
+  expect_is(b, "aligncol")
+  expect_identical(a$outer, "none")
```

Comment:
For completeness (and consistency with other assertions),  should there also be a `expect_identical(b$outer, "lr")`?